Homepage context

chloealee opened the pull request:
#### What's this PR do?
Adds more context about Rogue and of Team Bleed (!) to the homepage of Rogue. 

#### How should this be reviewed?
👀 

#### Any background context you want to provide?
@ngjo @lkpttn I kinda just winged the design - let me know if you want to change anything! 

Here's what it looks like: 
![screen shot 2017-11-07 at 4 03 56 pm](https://user-images.githubusercontent.com/9019452/32517774-6e5c7380-c3d5-11e7-8d75-e20b69ae1196.png)

#### Relevant tickets
Fixes this [Pivotal Card](https://www.pivotaltracker.com/n/projects/2019429/stories/152161310).

#### Checklist
- [ ] Documentation added for new features/changed endpoints.
- [ ] Tested on staging.
- [ ] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

ngjo commented:
@chloealee super minor, but can you capitalize 'mascot' - in terms of layout, I'll let @lkpttn provide feedback there!

chloealee commented:
oh yes @ngjo ! That is super annoying - updating now!

sbsmith86 commented on resources/views/pages/home.blade.php:
```diff
@@ -2,18 +2,43 @@
+                <div class="container__block -half">
+                    <ul class>
+                        <img src={{asset('images/Jen.png')}}>
+                        <li>Jen - Product Manager</li>
```
I think you should put the image and text in the `<li>` tag like:
```
<li>  
  <img />
  <p>text</p>
</li>
```

sbsmith86 commented on app/Http/routes.php:
```diff
@@ -107,4 +107,8 @@
     Route::delete('posts/{post}', 'Three\PostsController@destroy');
+
+    // reactions
+    Route::post('post/{post}/reactions', 'Three\ReactionController@store');
```
are these supposed to be added in this PR?

DFurnes commented on resources/views/pages/home.blade.php:
```diff
@@ -2,18 +2,57 @@
+                <h4>We're Team Bleed - your friends behind building and improving Rogue!</h4>
+                <div class="container__block -half">
+                    <ul class>
+                        <li>
```
These might work nicely as `.figure.-left.-small`s:

<img width="184" alt="screen shot 2017-11-07 at 1 50 07 pm" src="https://user-images.githubusercontent.com/583202/32511706-9557b2a0-c3c2-11e7-938f-fdef9cd7df2c.png">

sbsmith86 replied:
oooooo

chloealee replied:
ooooo yes!! Love this!!

chloealee replied:
@DFurnes updated this and updated the screenshot in the description! Great call!

DFurnes commented on resources/views/pages/home.blade.php:
```diff
@@ -2,18 +2,57 @@
     <div class="container -padded">
         <div class="wrapper">
-            <div class="container__block -narrow">
+            <div class="container__block">
```
It might be nice to make this a `.container__block.-narrow` so it's a bit easier to read.

lkpttn commented:
@chloealee We could do a real design pass on this later, but the only thing I'm thinking right now is to move the Log In button above the names and such. 

> Welcome to Rogue, blah blah blah
> [Log In]
> Team Bleed
> Contact @jen for access

DFurnes commented:
Oh another silly thought – if you want the [rounded avatars](http://forge.dosomething.org/#avatar) like we use elsewhere, adding the `.avatar` class to the pictures should do the trick. 🔵

chloealee commented:
@lkpttn that sounds good to me! Updated the screenshot again - let me know what you think!

chloealee commented:
@DFurnes ah yes - sorry I forgot about rounded! When I use `.avatar` class, it makes it a little bigger. Should I just add a custom css file here to make the image smaller?
![screen shot 2017-11-07 at 2 35 04 pm](https://user-images.githubusercontent.com/9019452/32513768-187cbdc8-c3c9-11e7-9e62-adcfc072ca7e.png)

DFurnes commented:
Ah yeah, looks like we did need a hack for that in Phoenix Next:

```css
.figure.-small > .figure__media img {
  width: 100%;
}
```

chloealee commented:
@DFurnes sorry I should know this but have been struggling for the last hour trying to add a custom css file to do this! I tried to do something like: 
```
@section('page-css')
    <link href="{{ asset('css/figure.css') }}" rel="stylesheet" type="text/css" >
@endsection
``` 
at the top of `home.blade.php` and then also add it to `<head>` section in `master.blade.php` but neither was working...

sbsmith86 commented:
@chloealee you could probably just add it in `app.scss` for now. We don't have a lot of custom styles in the app.

DFurnes commented:
Oh yeah, this is definitely fine to add to `app.scss`!

chloealee commented:
ah ok thanks @DFurnes and @sbsmith86 ! That still doesn't seem to work tho 🤔

sbsmith86 commented:
@chloealee can you post the code? Also just checking- did you rebuild the assets after making the change?

chloealee commented:
@sbsmith86 I did! Tried a few things:
```

// .figure.-small > .figure__media.avatar img {
  // width: 100%;
// }

.figure.-small > .figure__media.avatar {
  width: 50px;

  img { width: 100% }
}

// .figure.-small > .figure__media img {
//   width: 100%;
// }
```
Maybe the photos I'm using are just too small?

chloealee commented:
@sbsmith86 @DFurnes instead of using the `.avatar` class, I just changed the border radius to 100% to get the same look - lmk if this is ok! Updated screenshot up top.

DFurnes commented:
Looks perf! ⭕️

DFurnes commented on resources/assets/app.scss:
```diff
@@ -27,3 +27,7 @@ button.delete {
   border-bottom: 2px solid $purple;
 }
+
+.figure__media img {
```
Just want to make sure we're not using this anywhere else in this app, right?

DFurnes replied:
It might be a good idea to scope this to a `.rounded-figure` class just in case.

```scss
.figure-rounded .figure__media img {
 // ...
```

chloealee replied:
oh good call! @DFurnes looks like we're using it here: https://github.com/DoSomething/rogue/search?utf8=%E2%9C%93&q=figure__media&type=

But doesn't have an `img`. Just in case, should I add a `staff_pic` class to this too?

DFurnes replied:
Yeah, whatever class you want to use to scope the styles is fine. 👍

chloealee replied:
updated :) is this cool to merge now @DFurnes ?
